components/cloud-foundry/src/view/dashboard/cluster/actions/assign-users-workflow/assign-users.service.js

```diff
@@ -147,19 +148,19 @@
         allowJump: false,
         allowBack: true,
         allowCancelAtLastStep: true,
-        title: gettext('Assign User(s)'),
+        title: $translate.instant('assign-user-roles.title'),
```

Comment:
I think we can just pass in the translation id for titles and buttons, rather than needing to look them up here. There are a few places in this file, where I think the look-ups can be removed.